capstone/src/main/java/com/google/sps/data/Card.java

```diff
@@ -18,13 +18,18 @@
   private Integer id;
   private String front;
   private String back;
-  private Integer study_set_id; 
+  private Integer StudySetId; 
```

Comment:
First `S` should be lowercase: https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names